.github/workflows/ci.yml

```diff
@@ -112,6 +112,32 @@ jobs:
           go-version-file: go.mod
           cache: true
 
+      - name: Publish release on tag
+        uses: actions/github-script@v6
+        continue-on-error: true
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+              const ref = context.ref.split("/")[2]
+
+              const releases = (await github.rest.repos.listReleases({
+                owner: context.payload.repository.owner.login,
+                repo: context.payload.repository.name,
+                per_page: 100,
+              })).data
+
+              const draft_release = releases.find(release => release.draft && release.tag_name === ref)
```

Comment:
This looks great.  Can it be configured to start with v0.1.0?  It looks like this is incrementing the patch version to 0.0.1 for the first release.

Reply:
was this comment meant for the other file? There's no version in this file...

If so, it doesn't start from `v0.0.1`, it will just figure out from the labels that are in the changelog the appropriate version. If there are "features" it will increase the minor version, so `v0.1.0`, for breaking changes it will go to `v1.0.0` and so on following semver